Final project

brunoburzacca opened the pull request:
this is the final project incomplete

brunoburzacca committed:
no message

brunoburzacca committed:
this is the final project incomplete

gastonabastoG commented on .gitignore:
```diff
@@ -10,3 +10,512 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
```
Remember to use a better description in the commit

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/controller/IShoppingCart.java:
```diff
@@ -0,0 +1,17 @@
+public interface IShoppingCart {	 
+	
+	 public String registerUser(String name, String password, String mail, int phone);
+	 public String loggin(String name, String password);
```
log in is the correct spell, in this case you can use logIn

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/main/FindByCategory.java:
```diff
@@ -0,0 +1,48 @@
+			System.out.println("successful connection");
+
+			Statement stmt = conn.createStatement();
+			ResultSet rs = stmt.executeQuery("SELECT  *FROM product WHERE productCategory ='"+ category+ "'" );
```
It is better to use parameters

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/main/FindByName.java:
```diff
@@ -0,0 +1,47 @@
+			System.out.println("successful connection");
+
+			Statement stmt = conn.createStatement();
+			ResultSet rs = stmt.executeQuery("SELECT  *FROM product WHERE productName ='"+ name+ "'" );
```
It is better to use parameters

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/main/Product.java:
```diff
@@ -0,0 +1,40 @@
+	String producName=null, productCategory=null;
+	Double price;
+	
+	public Product(int productId1,String producName1,String productCategory1, Double price1){
```
Actually instead of using productId1, I would use productIdParam, and the same with the others fields

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/main/SaveCart.java:
```diff
@@ -0,0 +1,47 @@
+			 
+			 stmt.setString (1,billLines.getProductId() );
+			 stmt.setString (2, billLines.getCuantity());
+			 stmt.setInt (3, 1);
```
Will billId be always 1 ?

gastonabastoG commented on finalProject/src/main/java/org/globant/finalProject/main/UserLoggin.java:
```diff
@@ -0,0 +1,57 @@
+			System.out.println("successful connection");
+		
+			Statement stmt = conn.createStatement();
+			ResultSet rs = stmt.executeQuery("SELECT userName,userPassword,UserId FROM user");
```
Actually instead of get all user, and then iterate them to find the one that have the same name and password, you can create a select with where clause that handle these condition. If you get one record will mean that the user exist, if not the login will fail

brunoburzacca committed:
final project terminated

brunoburzacca committed:
there was an error when i push, so i have deleted the following classes:AddUser,App,BillLine,ConnectionManager,FindByCategory,FindByName,IShoppingCart,Product,SaveCart,UserLoggin.
Also i have correct the parameters error on classes cartDB,ProductDB,UserDB and Product

brunoburzacca committed:
Delete .gitignore

gitIgnore deleted
